examples/DynaSwing/SingleMachineSystem/SingleMachineSystem_ElectronicLoad/SingleMachineSystem.jobs

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!--
+    Copyright (c) 2023, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="Eurostag example - Fault">
+    <dyn:solver lib="dynawo_SolverIDA" parFile="SingleMachineSystem.par" parId="IDAOrder2"/>
+    <dyn:modeler compileDir="outputs/compilation">
+      <dyn:network iidmFile="SingleMachineSystem.iidm" parFile="SingleMachineSystem.par" parId="Network"/>
+      <dyn:dynModels dydFile="SingleMachineSystem.dyd"/>
+      <dyn:precompiledModels useStandardModels="true"/>
+      <dyn:modelicaModels useStandardModels="true"/>
+    </dyn:modeler>
+    <dyn:simulation startTime="0" stopTime="5"/>
+    <dyn:outputs directory="outputs">
```

Comment:
<dyn:timeline exportMode="TXT" filter="true"/>

This should be added if not, the timeline.log should be removed